### DISCUSSION TITLE
Does Organicmaps Supports fixed routing?

### DISCUSSION THREAD

**Discussion by sumitdua05** [organicmaps/organicmaps#3573] [Q&A]
I am trying to use OrganicMaps on Lubuntu . I have installed it using flatpak . Does this software supports fixed routing and traffic information?

**Comment by biodranik** [comment]
What is fixed routing? There is no traffic information yet. If you know good free sources of the traffic information, please let us know.

**Comment by sumitdua05** [comment]
By  Fixed routing, I meant route is already defined in some format which we can import and organic maps engine don't have to calculate any route path . It can follow that imported route and navigate on that route . Right now , I am not aware of any good free source . But, if we want to use a any paid version . Would it be easy to integrate Organic Maps with TomTom or Here for traffic information?

**Reply by biodranik** [reply]
Follow the imported route is planned, see #1360

You can try to contact TomTom and Here and ask if they can provide traffic info for free for Organic Maps, or at least for some feasible terms (e.g. only those users who use it are paying for it).
